src/components/emission/new/Form.tsx

```diff
@@ -28,32 +33,44 @@ const NewEmissionForm = () => {
     defaultValues: {
       name: '',
       attribute: '',
-      unit: '',
       source: '',
-      co2f: 0,
-      ch4f: 0,
-      ch4b: 0,
-      n2o: 0,
-      co2b: 0,
-      sf6: 0,
-      hfc: 0,
-      pfc: 0,
-      otherGES: 0,
+      co2f: [0],
+      ch4f: [0],
+      ch4b: [0],
+      n2o: [0],
+      co2b: [0],
+      sf6: [0],
+      hfc: [0],
+      pfc: [0],
+      otherGES: [0],
       totalCo2: 0,
       comment: '',
+      posts: [{ name: '', totalCo2: 0 }],
     },
   })
 
   const emissionValues = form.watch(['ch4b', 'ch4f', 'co2b', 'co2f', 'n2o', 'sf6', 'hfc', 'pfc', 'otherGES'])
+  const emissionPostsValues = form.watch('posts')
   const totalCo2 = form.watch('totalCo2')
   useEffect(() => {
     if (detailedGES) {
-      const newTotal = emissionValues.reduce((acc: number, current) => acc + (current || 0), 0)
+      const newTotal = emissionValues.reduce((acc: number, current) => {
+        const accountableValues = current?.filter((_, i) => (multipleEmissions ? i < postsCount : i === 0)) || []
+        const totalCurrent = accountableValues.reduce((acc: number, current: number) => acc + current, 0)
+        return acc + (totalCurrent || 0)
+      }, 0)
+      if (totalCo2 !== newTotal) {
+        form.setValue('totalCo2', newTotal)
+      }
+    } else {
+      const newTotal = (emissionPostsValues || [])
```

Comment:
emissionPostsValues est toujours defini non ?

Reply:
`'emissionPostsValues' is possibly 'undefined'`